README.md

```diff
@@ -112,7 +112,7 @@ After you've written your function, you can simply deploy it from your local mac
 
 ## Cloud Run/Cloud Run on GKE
 
-Once you've written your function and added the Functions Framework to your `requirements.txt` file, all that's left is to create a container image. [Check out the Cloud Run quickstart](https://cloud.google.com/run/docs/quickstarts/build-and-deploy) for Python to create a container image and deploy it to Cloud Run. You'll write a `Dockerfile` when you build your container. This `Dockerfile` allows you to specify exactly what goes into your container (including custom binaries, a specific operating system, and more).
+Once you've written your function and added the Functions Framework to your `requirements.txt` file, all that's left is to create a container image. [Check out the Cloud Run quickstart](https://cloud.google.com/run/docs/quickstarts/build-and-deploy) for Python to create a container image and deploy it to Cloud Run. You'll write a `Dockerfile` when you build your container. This `Dockerfile` allows you to specify exactly what goes into your container (including custom binaries, a specific operating system, and more). [Here is an example `Dockerfile` that calls Functions Framework.](./examples/cloud_run_http/Dockerfile)
```

Comment:
This made me realize that these relative links are broken [on PyPI](https://pypi.org/project/functions-framework/) and that they should be regular links instead (e.g. to https://github.com/GoogleCloudPlatform/functions-framework-python/blob/master/examples/cloud_run_http/Dockerfile).

Would you mind updating this as well as any other relative links in this README?

Reply:
Fixed